src/shared/config/pouchdb.ts

```diff
@@ -63,6 +63,11 @@ export const schema = [
     plural: 'medications',
     relations: { patient: { belongsTo: 'patient' } },
   },
+  {
```

Comment:
I don't know if this is necessary given that we don't know what pouchdb.ts will look like.

Reply:
Deleting it prevented the search function from working in InventoryRepository. I'm guessing it's because the file imports relationalDb from puchdb.ts which requires the use of the schema. I'm not sure what other way to do it since InventoryRepository requires some sort of database either way. The rest of the repositories seem to do it this way so I figure it's safe to do so as well.